packages/gdl-frontend/config.js

```diff
@@ -14,8 +14,10 @@ const graphqlEndpoint = () => {
   switch (GDL_ENVIRONMENT) {
     case 'dev':
       return 'http://localhost:4000';
+    case 'prod':
+      return 'https://api.digitallibrary.io/graphql-book-service';
```

Comment:
Ser dette riktig ut?

Reply:
ja, den adressen blir riktig i `prod`. Men jeg lurer på om vi skal sette `dev` til å gå mot test, grunnen er at jeg tenker at når man merger denne pull-requesten så er den avhengig av noe annet, så for at hvilken som helst utvikler skal kunne teste det lett så burde man slippe å kjøre det opp lokalt (med mindre man ønsker det :) ). Da kan man heller, lokalt, endre adressen til localhost. Så ville ha lagt til:

```javascript
 case 'local':
      return 'http://localhost:4000';
```
Og så satt dev til å gå mot `test` samt deployet `graphql-book.service` til `test`.

Reply:
Høres smart ut!